src/test/java/com/zollingpaper/backend/board/controller/BoardControllerE2ETest.java

```diff
@@ -0,0 +1,68 @@
+package com.zollingpaper.backend.board.controller;
+
+import static org.hamcrest.Matchers.is;
+
+import com.zollingpaper.backend.board.domain.BoardFixture;
+import com.zollingpaper.backend.board.exception.BoardErrorCode;
+import com.zollingpaper.backend.global.DatabaseCleaner;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+class BoardControllerE2ETest {
+
+    @LocalServerPort
+    int serverPort;
```

Comment:
```suggestion
private int serverPort;
```

접근제어자 private으로 해도 될 것 같습니다!